exam/demo/exam_demo.xml

```diff
@@ -181,6 +182,8 @@
             <field name="exam_date" eval="time.strftime('%Y')+'/08/20'"/>
             <field model="subject.subject" name="subject_id" search="[('name', '=', u'Science')]"/>
         </record>
+         -->
+         
```

Comment:
explain the reason to comment. Rather, fix it.

Reply:
These issues are fixed after my PR are merged.